eng/init-docker-compose.sh

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env bash
+set -e -o pipefail
+
+# SPDX-License-Identifier: Apache-2.0
+# Licensed to the Ed-Fi Alliance under one or more agreements.
+# The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+# See the LICENSE and NOTICES files in the project root for more information.
+
+# Switch to the Meadowlark-js directory
+pushd $(dirname "${BASH_SOURCE[0]}")/../Meadowlark-js
+{ # simulate `try`
+
+  # Build the local image
+  docker=docker
+  if [[ $(grep Microsoft /proc/version) ]]; then
+    # Running under WSL1, need to reference docker.exe instead of docker. Simple
+    # alias doesn't work, sadly. This variable helps in the context of this
+    # script, but need to use "wsl1" commands with npm.
+    docker=docker.exe
+    npm run docker:build:wsl1
+  else
+    npm run docker:build
+  fi
+
+  # Create a `.env-docker` file, injecting a new OAUTH signing key
+  # and random passwords for each container
+  echo "Creating .env-docker file..."
+  touch .env-docker
```

Comment:
If the .env-docker file already exists, the script inserts the values again. This is not an issue but could result in confusing behavior.

Reply:
In theory one should only be running this once. Of course you and I have run many times 😆 . I'm afraid of just deleting the file. Perhaps if there is already a file, we could error out with a warning. Then make the user deal with it.

Reply:
I added a question prompting the user if the file should be removed.